omnibus/cookbooks/omnibus-supermarket/Gemfile

```diff
@@ -8,7 +8,7 @@ gem 'chef', '~> 16.14'
 gem 'chef-bin', '~> 16.14'
 gem 'inspec-core'
 gem 'inspec-core-bin'
-gem 'mini_racer', '~> 0.3.1', platforms: :ruby
+# gem 'mini_racer', '~> 0.4.0', platforms: :ruby
```

Comment:
@RajeshPaul38 In this line we are first updating the version of `mini_racer` and then commenting the line. Can you please provide some context here?

Reply:
Yeah mini_racer won't be needed I believe. We have got rid of it from throughout the codebase. Will remove this line.

Reply:
Pushed the changes suggested.